docs/REST API/rest-api.md

```diff
@@ -5326,6 +5326,8 @@ If `mdm_id`, `mdm_name`, `mdm_enrollment_status`, `os_settings`, or `os_settings
 
 ### Delete label
 
+<!-- Return the following error if the user attempts to delete a label that's being used as a custom target for software: Software uses this label as a custom target. Please delete the software and try again. -->
+
```

Comment:
@noahtalerman just to double check, the[ UI spec has "Couldn't delete."](https://www.figma.com/design/4pfUOYy7IyMIrjMH2fuCdU/%2319551-Automatically-install-software-and-scope-software-with-labels?node-id=5493-9746&t=dyxeJtJBwn2T6DYX-1), so that should be added by the UI, and won't be part of the API error message? Or do we want them to match.

Reply:
Hmm, I think it makes sense to the API error message to match.

I updated the error message in the PR.

FYI @lucasmrod